static/hosts.html

```diff
@@ -0,0 +1,28 @@
+<html>
+  <head>
+    <title>Sia Explorer Alpha - Hosts</title>
+		<script src="static/third_party/jquery-2.1.4.min.js"></script>
+		<script src="static/third_party/URI.js"></script>
+		<script src="static/hosts.js"></script>
+		<link rel="stylesheet" href="static/hosts.css" type="text/css">
+  </head>
+  <body>
+    <h1>Sia Explorer Alpha - Hosts</h1>
+    <table>
+      <tr>
+				<td>Number of Hosts</td>
+				<td id="hostCount"></td>
+      </tr>
+    </table>
+    <table id="hostsTable">
+			<thead>
+				<tr>
+					<th id="addrHeader"><a href="#">Host Address</a></th>
+					<th id="storageHeader"><a href="#">Host Storage</a></th>
+					<th id="priceHeader"><a href="#">Price (per block per month)</a></th>
```

Comment:
Ah, I only got that from snippets of conversation in slack. I thought files were broken up into "blocks" (or chunks) and the price was per block of data per month, but I now understand block refers to blockchain block.

What are the units on `HostSettings.Price`? Is it price per GB/month?